hubstorage/client.py

```diff
@@ -135,7 +146,9 @@ def _create_retrier(self, max_retries, max_retry_time):
 
     def _create_session(self):
         s = session()
-        s.headers.update({'User-Agent': self.USERAGENT})
+        s.headers.update({'User-Agent': self.USERAGENT or self.user_agent})
```

Comment:
if `self.USERAGENT` is not empty (and we agreed that `USERAGENT = DEFAULT_USER_AGENT`) `self.user_agent` will not be used.